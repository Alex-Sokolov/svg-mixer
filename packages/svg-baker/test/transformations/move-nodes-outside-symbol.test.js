const { moveNodesOutsideSymbol } = require('svg-baker/lib/transformations');

const t = utils.testPlugin(moveNodesOutsideSymbol);

it('Move all gradients, patterns and filters outside symbol', async () => {
  expect(await t(
    undefined,
    '<svg><symbol><defs><linearGradient></linearGradient></defs></symbol></svg>',
  )).toMatchSnapshot();
});
