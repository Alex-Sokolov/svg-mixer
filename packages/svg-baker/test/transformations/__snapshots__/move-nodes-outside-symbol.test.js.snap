// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Move all gradients, patterns and filters outside symbol 1`] = `
"<svg>
  <symbol>
    <defs></defs>
  </symbol>
  <linearGradient></linearGradient>
</svg>"
`;
